### ISSUE TITLE
sudo_local in Sonoma

### ISSUE THREAD

**Issue by mwarkentin** [artginzburg/sudo-touchid#18]
Apple seems to have added a way for Touch ID to be enabled permanently in Sonoma:

https://www.idownloadblog.com/2023/08/24/touch-id-sudo-command-terminal-tutorial/

Should this be incorporated into this tool?

**Comment by artginzburg** [comment]
Yes, absolutely. The script will then inform the users that after usage, there's no need to keep the tool installed (other than for disabling pam_tid).

The script should just check the system version, if it's >=14.0 — do as described in the article instead of the old way.

Just installed Sonoma, so hopefully will have the time to modify the script soon.

> P.S. It seems like someone in Apple is subscribed to this repo :)

**Comment by artginzburg** [comment]
Done in #21
